splashsurf_lib/src/lib.rs

```diff
@@ -118,7 +118,7 @@ impl<R: Real> Parameters<R> {
 }
 
 /// Result data returned when the surface reconstruction was successful
-#[derive(Clone, Debug)]
+#[derive(Debug)]
```

Comment:
Is there a particular reason why you removed the Clone? Not that it's super important, but I think we don't have to prevent the user from cloning this?